app/components/supplier_table_component.html.haml

```diff
@@ -17,12 +23,6 @@
         %th.supplier-table__guarantee{ scope: "col" }
           .cads-table__content
             Customer commitments
-        %th{ scope: "col" }
-          .cads-table__content
-            .cads-hide-md-up
-              Rating
-            .cads-show-md-up
-              Overall rating
     %tbody.supplier-table__body
       - suppliers.each do |supplier|
         = render SupplierTableRowComponent.new(supplier, highlight: supplier.slug == highlight_supplier_with_slug, current_country:)
```

Comment:
I'm not very familiar with how the table works, but I'd imagine that the Table row component also needs to be updated to reflect the new order - at the moment I think the row below 'Overall rating' contains information about 'Fewer complaints received' and so on. 

The stars and more details link also need to be applied to the 'Overall rating' cell.

Reply:
Oh I get what you're saying now, the heading has moved position but the column itself hasn't. I'll sort that